.circleci/docker/common/install_rocm.sh

```diff
@@ -117,23 +127,46 @@ install_centos() {
       echo "gpgkey=http://repo.radeon.com/rocm/rocm.gpg.key" >> /etc/yum.repos.d/amdgpu.repo
   fi
 
-  local rocm_baseurl="http://repo.radeon.com/rocm/yum/${ROCM_VERSION}"
+  if [[ $OS_VERSION == 9 ]]; then
+      local rocm_baseurl="http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-centos-9/compute-rocm-dkms-no-npi-hipclang-10378"
+  else
+      local rocm_baseurl="http://repo.radeon.com/rocm/yum/${ROCM_VERSION}/main"
+  fi
   echo "[ROCm]" > /etc/yum.repos.d/rocm.repo
   echo "name=ROCm" >> /etc/yum.repos.d/rocm.repo
   echo "baseurl=${rocm_baseurl}" >> /etc/yum.repos.d/rocm.repo
   echo "enabled=1" >> /etc/yum.repos.d/rocm.repo
   echo "gpgcheck=1" >> /etc/yum.repos.d/rocm.repo
   echo "gpgkey=http://repo.radeon.com/rocm/rocm.gpg.key" >> /etc/yum.repos.d/rocm.repo
 
-  yum update -y
-
-  yum install -y \
+  if [[ $OS_VERSION == 9 ]]; then
+      yum update -y --nogpgcheck
+      dnf --enablerepo=crb install -y perl-File-BaseDir
+      yum install -y --nogpgcheck \
```

Comment:
`yum install -y --nogpgcheck rocm-ml-sdk rocm-developer-tools`
Should be enough to install rocm fully.